ui/src/Components/SearchCandidates.jsx

```diff
@@ -46,7 +46,7 @@ export default class SearchCandidates extends React.Component {
 	populate = (items, obj, i)=> {
 		items.push(<tr>
 			<th scope="row">{i+1}</th>
-			<td ><a href={"/profiles/"+obj[i].profile_id}>{obj[i].first_name} {obj[i].last_name}</a></td>
+			<td ><a href={"/profiles/"+obj[i].profile_id+"?admin=1"}>{obj[i].first_name} {obj[i].last_name}</a></td>
```

Comment:
Does this mean anybody can click on a profile from the search page and it'd automatically change you to "admin" mode?

Reply:
yes - but search based on MVP is not supposed to be accessed by anyone but the admin